lms/static/js/courseware/toggle_element_visibility.js

```diff
@@ -1,51 +0,0 @@
-(function(define) {
```

Comment:
Could you please clarify why `toggle_element_visibility` is removed?

Reply:
returned